starter-code/css/main.css

```diff
@@ -0,0 +1,126 @@
+body {
+  border-left:5px #ccc solid;
+  margin:0 0 0 5px;
+  font-family: Lato;
+}
+
+header {
+  max-width:700px;
+  padding-left:24px;
+}
+
+article {
+  max-width:700px;
+  padding-left:24px;
+}
+
+footer {
+  padding-left:24px;
+  max-width:700px;
+  border-top:1px #ccc solid;
+  padding-top:25px;
+}
+article h3{
+  color:#ccc;
+  font-weight:100;
+}
+
+img {
+  max-width: 100%;
+  max-height: 100%;
```

Comment:
You shouldn't have to set the height and width for an image, otherwise it could lead to stretching and rendering the image wonky. By using just the max-width, the height will adjust appropriately.